src/Toolbar/Toolbar.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ElementType, FC, HTMLAttributes } from 'react';
+import classnames from 'classnames';
+
+export interface ToolbarProps extends HTMLAttributes<HTMLElement> {
+  tag?: ElementType;
+  isMedium?: boolean;
+  isSmall?: boolean;
+}
+
+export const Toolbar: FC<ToolbarProps> = ({
+  tag = 'nav',
+  isMedium = false,
+  isSmall = false,
+  children,
+  ...attributes
+}) => {
+  const Tag = tag;
+  const classes = classnames('toolbar', {
+    'toolbar-medium': isMedium,
+    'toolbar-small': isSmall
+  });
```

Comment:
With the change above this would change in:
```suggestion
  const classes = classnames('toolbar', {
    [`toolbar-${size}`]': size
  });
```

Reply:
Same here: if the class 'toolbar-large' needed or it is by default.

Reply:
```js

const classes = classnames('toolbar', {
    [`toolbar-${size}`]': size != “large”
  });

```

this should work

Reply:
I did a little different: toolbar-size can be only 'medium' or 'small'. That is almost the same, except that classes which could be added to css can be only 'toolbar-medium' and 'toolbar-small' and not any others.